drop/modules/mae-pipeline/MAE/filterSNVs.sh

```diff
@@ -22,13 +22,25 @@ samtools=$8
 tmp=$(mktemp)
 
 echo 'Filter SNVs'
+
+# if not doing QC
```

Comment:
instead of 'if not doing QC', explain what is about to happen, or why is it needed